.github/workflows/build-binaries.yml

```diff
@@ -126,7 +122,7 @@ jobs:
           args: --release --locked --out dist --features self-update
       - name: "Test wheel - aarch64"
         run: |
-          pip install dist/${{ env.PACKAGE_NAME }}-*.whl --force-reinstall
+          pip install ${{ env.PACKAGE_NAME }} --no-index --find-links dist/ --force-reinstall
```

Comment:
Why prefer `--find-links` over installing the wheel directly?

Reply:
Currently on main, we have 6x `--find-links` and 4x `${{ env.PACKAGE_NAME }}-*` in `build-binaries.yml`. I prefer the `--find-links` as I can run it locally on any platform and shell, it doesn't rely on shell expansions.